packages/@markuplint/ml-core/src/ml-rule/ml-rule.ts

```diff
@@ -16,6 +16,14 @@ export class MLRule<T extends RuleConfigValue, O = null> {
 	#v: MLRuleOptions<T, O>['verify'];
 	#f: MLRuleOptions<T, O>['fix'];
 
+	get v() {
```

Comment:
@JounQin 

Is this necessary for accessing from the subclass?

Reply:
Yes, `#v` is inaccessible, it is [`hard private`](https://github.com/tc39/proposal-private-fields/issues/33).

Reply:
@JounQin 

Would you write a comment like "Unused internally, for 3rd party"?

Reply:
@YusukeHirao Done!